packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx

```diff
@@ -138,7 +141,10 @@ const ProposalForm = (): JSX.Element => {
     const proposalOptions = buildProposalOptions(values);
     const createProposalTx = await governor.buildCreateProposalTx(proposalOptions);
 
-    await sendSignAndPostRequest(connection, createProposalTx);
+    const transactionId = await sendSignAndPostRequest(connection, createProposalTx);
+    if (transactionId) {
+      dispatch(setTransactionsStateAction(transactionId));
```

Comment:
This should also be called for `transactionId === null` because then we know that `lastSignAndPostResult` was a rejection